### DISCUSSION TITLE
How to disable CodeMirror plugin on mobile

### DISCUSSION THREAD

**Discussion by saccarosium** [TiddlyWiki/TiddlyWiki5#7373] [Q&A] [answered]
I've installed the CodeMirror editor because I wand to use vim keybinding in TW, but when I use my wiki on my phone, most normal phone editing functions don't work, for example selecting text and copying it on IOS.

I've serached already open or closed issues and I found [this one](https://github.com/Jermolene/TiddlyWiki5/issues/2730), where it seems that the issue is fixed. But as a TW noob I don't really know were I need to put the code to make it work.

Any help?

**Comment by pmario** [comment, marked as answer]
**Backup first**

- Take the code posted at the thread starting with `(function(){`
- Copy it into a tiddler named: `$:/core/modules/macros/hackeditorstring.js`
- Set the tiddler type to: `application/javascript`
- Create a field: `module-type` and set the value to: `startup`
- Save the tiddler
- reload your wiki.

That should do it.

**Reply by saccarosium** [reply]
Doesn't seems to work. The name of the tiddler is literally `title: $:/core/modules/macros/hackeditorstring.js` instead of `$:/core/modules/macros/hackeditorstring.js`?

**Reply by pmario** [reply]
sorry copy / paste problem. Did update the post. 
Did you also test the second code snippet?

**Reply by saccarosium** [reply]
I've tested only the second snippet and it work on my phone but not on my ipad, but I think that my question was asked. Thank you!
